dockerfiles/nanocloud-backend/Dockerfile

```diff
@@ -11,6 +11,23 @@ COPY repos_nanocloud /opt/repos
 
 RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && apt-get install -y nodejs
 
+RUN npm install -g bower
```

Comment:
To avoid creating to many Docker layers, npm install should fit in one command. Like : 
RUN npm install -g grunt-cli \
grunt \
....